# -*- coding: utf-8 -*-

from PySide2 import QtCore, QtGui, QtWidgets

class Ui_StockDialog:
    def setupUi(self, StockDialog):
        StockDialog.setObjectName("StockDialog")
        StockDialog.resize(1000, 700)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(StockDialog.sizePolicy().hasHeightForWidth())
        StockDialog.setSizePolicy(sizePolicy)
        StockDialog.setMinimumSize(QtCore.QSize(1000, 700))
        self.gridLayout = QtWidgets.QGridLayout(StockDialog)
        self.gridLayout.setObjectName("gridLayout")
        self.back_frame = QtWidgets.QFrame(StockDialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.back_frame.sizePolicy().hasHeightForWidth())
        self.back_frame.setSizePolicy(sizePolicy)
        self.back_frame.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.back_frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.back_frame.setObjectName("back_frame")
        self.gridLayout_3 = QtWidgets.QGridLayout(self.back_frame)
        self.gridLayout_3.setObjectName("gridLayout_3")
        self.table_frame = QtWidgets.QFrame(self.back_frame)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.table_frame.sizePolicy().hasHeightForWidth())
        self.table_frame.setSizePolicy(sizePolicy)
        self.table_frame.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.table_frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.table_frame.setObjectName("table_frame")
        self.gridLayout_2 = QtWidgets.QGridLayout(self.table_frame)
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.stock_table = QtWidgets.QTableWidget(self.table_frame)
        self.stock_table.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.stock_table.setAutoFillBackground(False)
        self.stock_table.setSizeAdjustPolicy(QtWidgets.QAbstractScrollArea.AdjustToContents)
        self.stock_table.setAutoScrollMargin(18)
        self.stock_table.setTextElideMode(QtCore.Qt.ElideMiddle)
        self.stock_table.setShowGrid(True)
        self.stock_table.setGridStyle(QtCore.Qt.SolidLine)
        self.stock_table.setWordWrap(False)
        self.stock_table.setRowCount(0)
        self.stock_table.setColumnCount(5)
        self.stock_table.setObjectName("stock_table")
        self.stock_table.setColumnCount(5)
        self.stock_table.setRowCount(0)
        item = QtWidgets.QTableWidgetItem()
        self.stock_table.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.stock_table.setHorizontalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.stock_table.setHorizontalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.stock_table.setHorizontalHeaderItem(3, item)
        item = QtWidgets.QTableWidgetItem()
        self.stock_table.setHorizontalHeaderItem(4, item)
        self.stock_table.horizontalHeader().setVisible(True)
        self.stock_table.horizontalHeader().setCascadingSectionResizes(False)
        self.stock_table.verticalHeader().setMinimumSectionSize(17)
        self.gridLayout_2.addWidget(self.stock_table, 0, 0, 1, 1)
        self.gridLayout_3.addWidget(self.table_frame, 0, 0, 1, 1)
        self.gridLayout.addWidget(self.back_frame, 0, 0, 1, 1)

        self.retranslateUi(StockDialog)
        QtCore.QMetaObject.connectSlotsByName(StockDialog)

    def retranslateUi(self, StockDialog):
        StockDialog.setWindowTitle(QtWidgets.QApplication.translate("StockDialog", "StockDialog", None, -1))
        self.stock_table.horizontalHeaderItem(0).setText(QtWidgets.QApplication.translate("StockDialog", "Book ID", None, -1))
        self.stock_table.horizontalHeaderItem(1).setText(QtWidgets.QApplication.translate("StockDialog", "Title", None, -1))
        self.stock_table.horizontalHeaderItem(2).setText(QtWidgets.QApplication.translate("StockDialog", "Author", None, -1))
        self.stock_table.horizontalHeaderItem(3).setText(QtWidgets.QApplication.translate("StockDialog", "Price", None, -1))
        self.stock_table.horizontalHeaderItem(4).setText(QtWidgets.QApplication.translate("StockDialog", "Quantity", None, -1))

